File: bnu_lms_app/lib/features/ai_chat/data/models/send_message_request.dart
class SendMessageRequest {
  final int? sessionId;
  final int? courseId;
  final String content;
  final String? base64Image;

  SendMessageRequest({
    this.sessionId,
    this.courseId,
    required this.content,
    this.base64Image,
  });

  Map<String, dynamic> toJson() {
    return {
      'sessionId': sessionId,
      'courseId': courseId,
      'content': content,
      'base64Image': base64Image,
    };
  }
}

File: backend/application/Dtos/AI/SendMessageRequest.cs
namespace CampusConnect.Application.Dtos.AI
{
    public class SendMessageRequest
    {
        public int? SessionId { get; set; }
        public int? CourseId { get; set; }
        public string Content { get; set; }
        public string? Base64Image { get; set; }
    }
}
